cg/store/api/status.py

```diff
@@ -88,17 +88,19 @@ def get_families_with_samples(self) -> Query:
         return self.Family.query.join(Family.links, FamilySample.sample, Family.customer)
 
     def cases_to_analyze(
-        self, pipeline: Pipeline = None, threshold: bool = False, limit: int = None
+        self, pipeline: str = None, threshold: bool = False, limit: int = None
```

Comment:
Is this typehint change intended? For me  it's more explanatory to keep using `:Pipeline`

Reply:
It was a mix of str and Pipeline. Now we just use Pipeline 👍 . Fixed https://github.com/Clinical-Genomics/cg/pull/1754/commits/8d5ebcf23d1ceb8f1fbd0ccd7d06bd107efd0f9e